include/eve/module/core/regular/impl/simd/arm/neon/minmax.hpp

```diff
@@ -0,0 +1,30 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/concept/value.hpp>
+#include <eve/detail/apply_over.hpp>
+#include <eve/detail/implementation.hpp>
+
+namespace eve::detail
+{
+// -----  Regular
+template<arithmetic_scalar_value T, typename N>
+EVE_FORCEINLINE auto
+minmax_(EVE_SUPPORTS(neon128_), wide<T, N> v0, wide<T, N> v1) noexcept requires arm_abi<abi_t<T, N>>
+{
+  if constexpr(sizeof(T) == 8)
+  {
+    // There is no min/max for 64bits type, so we compute the check once
```

Comment:
I thought you assume this is optimized. Otherwise you should do it everywhere.

Reply:
I checked all codegen. Neon requires, SVE is fine, PPC is fine.